From d19d61508216bfa0e6b5b391a920d78092284731 Mon Sep 17 00:00:00 2001
From: RolandoMagico <75312877+RolandoMagico@users.noreply.github.com>
Date: Sat, 24 Jun 2023 20:17:47 +0200
Subject: [PATCH] Update OpenWrt version

---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 333b29b2..fefd5029 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing gluon'
 
 OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-22.03
-OPENWRT_COMMIT=e500494771537b9f42f78e4d907bed18b6383606
+OPENWRT_COMMIT=2034387af45f11c933e36ebdde4ca198c45068f9
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-22.03
-- 
2.41.0.windows.1

