diff --git a/modules b/modules
index 333b29b2..4e688d3d 100644
--- a/modules
+++ b/modules
@@ -2,11 +2,11 @@ GLUON_FEEDS='packages routing gluon'
 
 OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-22.03
-OPENWRT_COMMIT=e500494771537b9f42f78e4d907bed18b6383606
+OPENWRT_COMMIT=a7867e0cb74c887ac6f3e0af4c97447114d382bc
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-22.03
-PACKAGES_PACKAGES_COMMIT=55eed1761207f4dfdb8e7d79138f6f65c8849b50
+PACKAGES_PACKAGES_COMMIT=6d28f29464de5988f2688e467e038c4b160a0551
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-22.03
