diff --git a/targets/ramips-mt7621 b/targets/ramips-mt7621
index 3337b265..85de2543 100644
--- a/targets/ramips-mt7621
+++ b/targets/ramips-mt7621
@@ -6,6 +6,10 @@ device('asus-rt-ac57u', 'asus_rt-ac57u', {
 
 
 -- D-Link
+device('d-link-covr-x1860-a1', 'dlink_covr-x1860-a1', {
+	factory_ext = '-recovery.bin',
+})
+
 device('d-link-dap-x1860-a1', 'dlink_dap-x1860-a1')
 
 device('d-link-dir-860l-b1', 'dlink_dir-860l-b1')
